lib/ovirt/template.rb

```diff
@@ -152,9 +152,7 @@ def build_clone_xml(options)
             end
           end
 
-          xml.os(:type => options[:os_type] || os_type) do
-            xml.boot(:dev => 'hd')
-          end
+          xml.os(:type => options[:os_type] || os_type)
```

Comment:
@matobet During provisioning if you select a template to use as the base for PXE or ISO provisioning, I would expect all attributes (boot order included) to be inherited from this template.  Behind the scenes, since we use the blank template as the source of the clone, those values wouldn't be inherited by default and we need to set them.  The defaults would be inherited from the blank template instead of the selected template (which might have a different boot order).